lib/login.dart

```diff
@@ -107,46 +116,54 @@ class _LoginPageState extends State<LoginPage> {
 
   Widget createAccount(BuildContext context) {
     return FlatButton(
-      child: Text('create account'),
+      child: const Text('create account'),
       onPressed: () {
         Navigator.pushNamed(context, RecordingRoute);
       },
     );
   }
 
-  void makeLoginRequest() async {
-    String userEmail = loginController.text;
-    String url = 'http://localhost:9000/user/${userEmail}';
-    //String url = "http://10.0.2.2:9000/user/${userEmail}/";
-    var response = await http
-        .get(Uri.encodeFull(url), headers: {"Accept": "application/json"});
-    var userData = json.decode(response.body);
-    SharedPreferences prefs = await SharedPreferences.getInstance();
-    prefs.setString("name", userData['firstName'].toString());
-    setState(() {
-      userId = int.parse(userData['Id'].toString());
-    });
+  Future<void> makeLoginRequest() async {
+    final String userEmail = loginController.text;
+    final String url = 'http://localhost:9000/user/$userEmail';
+    //String url = 'http://10.0.2.2:9000/user/${userEmail}/';
+    final http.Response response = await http.get(Uri.encodeFull(url),
+        headers: <String, String>{'Accept': 'application/json'});
+    final dynamic userData = json.decode(response.body);
+    final SharedPreferences prefs = await SharedPreferences.getInstance();
+    prefs.setString(
+      'name',
```

Comment:
kind of looks like you're off a space here

Reply:
Can you clarify this?